README.md

```diff
@@ -1,3 +1,5 @@
+**DEPRECATED: All the functionality has been moved to [commerce-on-ligtning](https://github.com/forcedotcom/commerce-on-lightning) repository.**
```

Comment:
typo: `commerce-on-ligtning` --> `commerce-on-lightning`

Reply:
fixed it.